packages/shared_preferences/shared_preferences_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+* Adds `package:shared_preferences_macos`.
```

Comment:
```suggestion
* Increase minimum Flutter SDK to 1.12.8.
```

Would it make sense to do this change in a separate PR?

Reply:
This was required by the CI tests. Failing if the version wasn't increased.

Reply:
Sorry, another PR was landed before this one, changing the version and changelog covering your suggestion. I still had to bump the version and the only change was adding the macos package.